external/external.sh

```diff
@@ -19,6 +19,33 @@ set -e
 
 source $(dirname "$0")/provider.sh
 
+if [ -z "${EXTRA_DOCKER_ARGS}" ] ; then
+  echo \
+"   # ================= WARNING ================= WARNING ================= WARNING ================= #
+   # EXTRA_DOCKER_ARGS are not set. You will be testing java which is already in container and not TEST_JDK_HOME
+   # TEST_JDK_HOME is set to $TEST_JDK_HOME but will not be used. See test_base_functions.sh for order of search
+   # You should set your's TEST_JDK_HOME to mount to /opt/java/openjdk, eg:                          #
+   # export EXTRA_DOCKER_ARGS=\"-v \$TEST_JDK_HOME:/opt/java/openjdk\"                               #
+   # ================= WARNING ================= WARNING ================= WARNING ================= #"
+else
+  echo \
+"   # =================== Info =================== Info =================== Info =================== #
+   # EXTRA_DOCKER_ARGS set as \"$EXTRA_DOCKER_ARGS\"                                #
+   # =================== Info =================== Info =================== Info =================== #"
+  if echo "${EXTRA_DOCKER_ARGS}" | grep -q "$TEST_JDK_HOME"  ; then
+    echo \
+"   # =================== Info =================== Info =================== Info =================== #
+   # TEST_JDK_HOME of $TEST_JDK_HOME is used in EXTRA_DOCKER_ARGS                                #
+   # =================== Info =================== Info =================== Info =================== #"
+  else
+    echo \
+"   # ================= WARNING ================= WARNING ================= WARNING ================= #
+   # TEST_JDK_HOME of $TEST_JDK_HOME is NOT used in EXTRA_DOCKER_ARGS                                #
+   # ================= WARNING ================= WARNING ================= WARNING ================= #"
+  fi
+fi
+
```

Comment:
For the record, not a great fan of this output.  My original review said to change the previous incarnation to look more professional.  Not going to block the PR on it, but think that this is excessively noisy and seems like its better suited for a debugging session than a production run.